joblib/memory.py

```diff
@@ -782,6 +782,18 @@ def _persist_input(self, duration, args, kwargs, this_duration_limit=0.5):
         return metadata
 
     # XXX: Need a method to check if results are available.
+    def check_call_in_cache(self, *args, **kwargs):
+        """
```

Comment:
Can you make the docstring follow the [numpy docstring convention](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)? Thanks!

Reply:
Fixed the docstring I think.